api-specificatie/openapi.yaml

```diff
@@ -123,11 +157,19 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/VolgindicatiePersonen"
+                $ref: "#/components/schemas/VolgindicatiePersoon"
         '204':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/204"
+        '401':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/401"
+        '403':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/403"
         '404':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/404"
+        '410':
```

Comment:
je maakt een aparte afhandeling wanneer je een volgindicatie probeert te verwijderen die al eerder verwijderd is. Is dat voor de gebruiker relevant anders dan een 404 (volgindicatie bestaat niet)?

Reply:
Eens! Hier gelden dezelfde argumenten als bij een GET /volgindicatiepersonen/{burgerservicenummer}.